### PR TITLE
fix(chat): Display text from image message

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: #9564

### What does the PR do

Mobile app sending images with text in one message. This PR adds filtering images in messages and displaying text from message.

P.S.: Don't know how to fix out placeholder in image messages which already sent.

### Affected areas

Chat

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

### CODE REVIEW

File: ui/StatusQ/src/StatusQ/Components/StatusMessage.qml [hunk 1]
```diff
@@ -274,10 +274,24 @@ Control {
...
                         active: root.messageDetails.contentType === StatusMessage.ContentType.Image && !editMode
                         visible: active
-                        sourceComponent: StatusImageMessage {
-                            source: root.messageDetails.messageContent
-                            onClicked: root.imageClicked(image, mouse, imageSource)
-                            shapeType: root.messageDetails.amISender ? StatusImageMessage.ShapeType.RIGHT_ROUNDED : StatusImageMessage.ShapeType.LEFT_ROUNDED
+                        sourceComponent: Column {
+                            spacing: 8
+                            Loader {
+                                active: root.messageDetails.messageText != ""
```
Comment (jrainville) [inline ui/StatusQ/src/StatusQ/Components/StatusMessage.qml:280]:
```suggestion
                                active: root.messageDetails.messageText !== ""
```

File: src/app/modules/main/chat_section/chat_content/messages/module.nim [hunk 2]
```diff
@@ -219,7 +221,11 @@ method newMessagesLoaded*(self: Module, messages: seq[MessageDto], reactions: se
       if chatDetails.communityId != "":
         communityChats = self.controller.getCommunityById(chatDetails.communityId).chats
 
-      let renderedMessageText = self.controller.getRenderedText(message.parsedText, communityChats)
+      var renderedMessageText = self.controller.getRenderedText(message.parsedText, communityChats)
+
+      if (prevMessage.contentType.ContentType == ContentType.Image and prevMessage.messageText == renderedMessageText):
+        renderedMessageText = ""
```
Comment (jrainville) [inline src/app/modules/main/chat_section/chat_content/messages/module.nim:227]: Can you explain why you do that?

Comment (borismelnik) [reply src/app/modules/main/chat_section/chat_content/messages/module.nim:227]: if someone send several images with text from mobile app, we receive it as several image messages with same text. I skip text from images for removing duplications and display this text just for last one.

Comment (jrainville) [reply src/app/modules/main/chat_section/chat_content/messages/module.nim:227]: Ah ok, got it. Could you add that as a comment here, that way we know what it does. Thanks

**Review by mprakhov — APPROVED** [review]
LGTM 👍

**Review by caybro — APPROVED** [review]
LGTM

**Review by jrainville — APPROVED** [review]

**Comment by anastasiyaig** [comment]
okay guys, it is merged to latest master, i pulled it and this is the result of sending picture+text:

sending a picture with text still results in 2 messages, when it should be 1 message. I am on commit 71bbb03faa2d41d460dab16b7bc2f68f1d4c73db. Was it tested before merge? 

Screenshot 2023-03-03 at 13 55 29

**Comment by caybro** [comment]
> okay guys, it is merged to latest master, i pulled it and this is the result of sending picture+text:
> 
> sending a picture with text still results in 2 messages, when it should be 1 message. I am on commit 71bbb03faa2d41d460dab16b7bc2f68f1d4c73db. Was it tested before merge? 
> 
> Screenshot 2023-03-03 at 13 55 29
> 

Does the image also have some extra emoji?

**Comment by anastasiyaig** [comment]
nope, just pure image with text

**Comment by anastasiyaig** [comment]
this is example: image with text results in 2 messages and gif+text 

Screenshot 2023-03-03 at 14 21 00

**Comment by saledjenic** [comment]
@anastasiyaig I see now that one of your contacts has 3 words name, is that an old contact (or old Status account) or some bug in the latest master?

**Comment by anastasiyaig** [comment]
@saledjenic it is contact generated on mobile device, they are still using 3 words names

**Comment by caybro** [comment]
> > okay guys, it is merged to latest master, i pulled it and this is the result of sending picture+text:
> > sending a picture with text still results in 2 messages, when it should be 1 message. I am on commit [71bbb03](https://github.com/status-im/status-desktop/commit/71bbb03faa2d41d460dab16b7bc2f68f1d4c73db). Was it tested before merge?
> > Screenshot 2023-03-03 at 13 55 29
> 
> Does the image also have some extra emoji?

I see a little "image" emoji above the actual image

**Comment by jrainville** [comment]
@anastasiyaig can you open a simple issue and assign to @borismelnik please? Thanks

**Comment by anastasiyaig** [comment]
i just reopened initial one @jrainville

**Pull request merged**